packages/database/src/migrations/20200914080738-AddLaosSchoolsSchoolLevelICTFacilitiesDashboardReport-modifies-data.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+import { insertObject } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DASHBOARD_GROUP_CODE = 'LA_Laos_Schools_School_Laos_Schools_User';
+
+const REPORT_ID = 'Laos_Schools_ICT_Facilities';
+
+const DATA_BUILDER_CONFIG = {
+  rows: [
+    'Electricity Available',
+    'Functioning TV for teaching and learning purposes',
+    'Functioning smart TV',
+    'Lao satellite receiver and dish set',
+    'Functioning computer',
+    'Functioning projector',
+  ],
+  cells: [
+    ['SchFF001', 'SchCVD011'],
+    ['SchCVD012'],
+    ['SchCVD012a'],
+    ['SchCVD012b'],
+    ['SchCVD013', 'SchCVD014'],
+    ['SchCVD015'],
+  ],
+  columns: ['Main', 'Extra'],
+  entityAggregation: {
+    dataSourceEntityType: 'school',
+  },
+};
+
+const VIEW_JSON = {
+  name: 'ICT Facilities',
+  type: 'matrix',
```

Comment:
Is it ok as a matrix or does it need to be viewable in the sidebar? The card says "Binary Table with Response column".

Reply:
Ooopppsssss... I didn't read it carefully. Will confirm with Megan!